src/components/Breadcrumbs/Breadcrumbs.astro

```diff
@@ -18,12 +18,12 @@ const schema = {
     {
       '@type': 'ListItem',
       name: t('breadcrumbs_home'),
-      item: new URL(`/${ locale }/`, Astro.site),
+      item: new URL(`/${ locale }/`, Astro.site || 'http://localhost:5173/'),
```

Comment:
What is happening here? I see that Astro.site is undefined in Vitest. I can also verify that using `vi.stubGlobal` does not resolve this. Then I would expect the `renderFragment `in the `renderer.ts` to be able to use the Astro config, which is a valid property for the `AstroContainer.create()` function. But it doesn't seem to work atm unfortunately.

I would opt to remove the fallback here, because it can have unintended consequences, and skip the breadcrumbs component for now, with a TODO stating the issue